rfcs/0114-nixos-code-of-conduct.md

```diff
@@ -0,0 +1,108 @@
+---
+feature: NixOS Code of Conduct
+start-date: 2021-11-03
+author: @jonringer
+co-authors:
+shepherd-team:
+shepherd-leader:
+related-issues: #98 #102
+---
+
+# Summary
+[summary]: #summary
+
+The NixOS Community needs some foundation of shared values which can be used
+to determine appropriate when contributing and interacting with the community.
+We currently have a [community ethos](https://nixos.org/community/index.html), but
+this only states that we are an inclusive community, but doesn't define
```

Comment:
```suggestion
this only states that we are an inclusive community, and doesn't define
```

Reply:
I'm terrible at proper english grammar, so I'm probably wrong. But I think `but` is more appropriate as I'm listing what it's doing, `and` to me should be used when doing another thing (not the absence thereof).